tests/integration/dual-channel-replication.tcl

```diff
@@ -1243,3 +1243,59 @@ start_server {tags {"dual-channel-replication external:skip"}} {
         stop_write_load $load_handle
     }
 }
+
+
+start_server {tags {"dual-channel-replication external:skip"}} {
+    set primary [srv 0 client]
+    set primary_host [srv 0 host]
+    set primary_port [srv 0 port]
+
+    $primary config set repl-diskless-sync yes
+    $primary config set dual-channel-replication-enabled yes
+    $primary config set repl-diskless-sync-delay 5; # allow catch failed sync before retry
+
+    # Generating RDB will take 100 sec to generate
+    $primary debug populate 1000000 primary 1
+    $primary config set rdb-key-save-delay -10
```

Comment:
Does this test need to take 100+ seconds? The dual channel tests are some of the worst offenders today for taking a lot of time to complete.

Reply:
@naglera can you please check if we can reduce that?

Reply:
To clarify, while the comment suggests it could take 100 seconds to generate the RDB, the actual test duration is likely much shorter. The master abort the RDB generation process when the replica's main connection is killed, rather than waiting for the full RDB generation to complete.
In theory, we could set this to an even larger number (e.g., 1,000,000 seconds) without significantly impacting the test duration, as the process should be interrupted well before completion.

If you're seeing that the dual channel tests are taking an unusually long time, it would be very helpful if you could share some logs or specific timing information.